class.js

```diff
@@ -15,18 +15,35 @@ var types = ["Fire", "Water", "Grass", "Electric", "Flying", "Poison", "Fairy",
 //there's probably a more efficient way to do this but that's for Future Liza to figure out
 
 //ok so for evolution: I'm going to have a list of all of the Pokemon. When trying to evolve you first check to see if it's a Pokemon that can evolve at all, and if so, then you find the indexOf the Pokemon name in the Pokemon name list, and then go one index further.
-var pkmList = ["Pikachu", "Raichu", "Charmander", "Charmeleon", "Charizard", "Squirtle", "Wartortle", "Blastoise", "Bulbasaur", "Ivysaur", "Venusaur", "Pidgey", "Pidgeotto", "Pidgeot", "Caterpie", "Metapod", "Butterfree", "Ekans", "Arbok", "Jigglypuff", "Wigglytuff", "Zubat", "Golbat", "Crobat", "Oddish", "Gloom", "Vileplume", "Doduo", "Dodrio", "Vulpix", "Ninetales", "Gastly", "Haunter", "Gengar", "Weedle", "Kakuna", "Beedrill", "Clefairy", "Clefable", "Meowth", "Persian", "Growlithe", "Arcanine"];
+var pkmList = [["Pikachu", "Electric"], ["Raichu", "Electric"], ["Charmander", "Fire"], ["Charmeleon", "Fire"], ["Charizard", "Fire"], ["Squirtle", "Water"], ["Wartortle", "Water"], ["Blastoise", "Water"], ["Bulbasaur", "Grass"], ["Ivysaur", "Grass"], ["Venusaur", "Grass"], ["Pidgey", "Flying"], ["Pidgeotto", "Flying"], ["Pidgeot", "Flying"], ["Caterpie", "Bug"], ["Metapod", "Bug"], ["Butterfree", "Bug"], ["Ekans", "Poison"], ["Arbok", "Poison"], ["Jigglypuff", "Fairy"], ["Wigglytuff", "Fairy"], ["Zubat", "Poison"], ["Golbat", "Poison"], ["Crobat", "Poison"], ["Oddish", "Grass"], ["Gloom", "Poison"], ["Vileplume", "Poison"], ["Doduo", "Normal"], ["Dodrio", "Normal"], ["Vulpix", "Fire"], ["Ninetales", "Fire"], ["Gastly", "Ghost"], ["Haunter", "Ghost"], ["Gengar", "Ghost"], ["Weedle", "Bug"], ["Kakuna", "Bug"], ["Beedrill", "Bug"], ["Clefairy", "Fairy"], ["Clefable", "Fairy"], ["Meowth", "Normal"], ["Persian", "Normal"], ["Growlithe", "Fire"], ["Arcanine", "Fire"], ["Magnemite", "Electric"], ["Magneton", "Electric"], ["Magnezone", "Electric"], ["Lapras", "Water"], ["Rattata", "Normal"], ["Raticate", "Normal"], ["Psyduck", "Water"], ["Golduck", "Water"], ["Mewtwo", "Psychic"]];
 
 var wildPokemon = [["Pidgey", "Flying"], ["Caterpie", "Bug"], ["Ekans", "Poison"], ["Jigglypuff", "Fairy"], ["Zubat", "Flying"], ["Oddish", "Grass"], ["Doduo", "Normal"], ["Vulpix", "Fire"], ["Gastly", "Ghost"], ["Rattata", "Normal"], ["Weedle", "Bug"], ["Clefairy", "Fairy"], ["Meowth", "Normal"], ["Growlithe", "Fire"]]; //list of wild Pokemon to grab from for a random encounter
 
-var trainerList = [["Kyle", "Rattata", "Normal", 5], ["Emma", "Oddish", "Grass", 10], ["Kovas", "Golduck", "Water", 20], ["Cora", "Growlithe", "Fire", 25], ["Basilio", "Magnemite", "Electric", 30], ["Isabel", "Ninetales", "Fire", 40], ["Amy", "Haunter", "Ghost", 45], ["Zoya", "Lapras", "Water", 50], ["Albinson", "Mewtwo", "Psychic", 100]]; 
```

Comment:
Similar to a comment I left below about `pkmList` the elemens of this list might benefit from being objects with properties rather than arrays that you have to take apart.